x/gov/keeper/msg_server_test.go

```diff
@@ -27,15 +27,12 @@ func (suite *KeeperTestSuite) TestSubmitProposalReq() {
 	addrs := suite.addrs
 	proposer := addrs[0]
 
-	coins := sdk.NewCoins(sdk.NewCoin("stake", sdkmath.NewInt(100000)))
+	coins := sdk.NewCoins(sdk.NewCoin("matic", sdkmath.NewInt(100000000000000000)))
 	initialDeposit := coins
 	params, _ := suite.govKeeper.Params.Get(suite.ctx)
 	minDeposit := params.MinDeposit
-	bankMsg := &banktypes.MsgSend{
```

Comment:
One thing I've noticed that most tests in this PR simply replace a bank msg send with a text proposal...would love to see a test ensuring that only the update params messages pass (worth an exhaustive test), as well as a bank send to ensure it gets rejected.

Reply:
Good point. I actually proved that when `bankMsg` was there and I was getting errors, but I'll make sure to implement it.

Reply:
I would leave it more generic just because theoretically you could create a proposal that doesn't just emit a Bank Send but also Delegating, though I am not sure if you will disable that message in the custom Stake module

Reply:
Also @marcello33 that commit 404's for me

Reply:
Yeah we won’t enable delegation in staking module 
Sorry, here’s the commit 
https://github.com/0xPolygon/cosmos-sdk/pull/7/commits/39fd8a993160d716f503d10c9f5567f397c4605f